h/static/styles/topbar.scss

```diff
@@ -41,7 +41,7 @@
 
     & > *:first-child, & > *:last-child {
       line-height: 28px;
-      margin: 0 .72em;
+      margin-right: .72em;
```

Comment:
Then you should adjust the CSS around media queries and the topbar to make this a symmetric change.